src/Files.App/Actions/Display/GroupAction.cs

```diff
@@ -567,7 +615,12 @@ public ToggleGroupByDateUnitAction()
 
 		public Task ExecuteAsync()
 		{
-			context.GroupByDateUnit = context.GroupByDateUnit is GroupByDateUnit.Month ? GroupByDateUnit.Year : GroupByDateUnit.Month;
+			context.GroupByDateUnit = context.GroupByDateUnit switch
```

Comment:
I dont think the `ToggleGroupByDateUnitAction` is currently used. I have changed it in case it will be used in the future.
Alternatively, it could also be removed completely.

Reply:
It should be accessible from the command palette

Reply:
thanks, you're right. i have just tested it